eng/Versions.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
Which one wins, this one, or the root Directory.Build.props? (My guess is that the root one wins, because I think it includes this then over-writes it).  Ideally this value is only defined once.

Reply:
it really depends on when the imports are happening. I agree with the suggestion that this should really only be defined once, and I would keep this one as this file should contain all the versioning properties, so I would remove the one from Directory.Build.props

Reply:
Done. Defined only in `eng/Versions.props` now. Also bumped the version by incrementing the patch by 1 and renamed the packages to have `alpha` now instead of `preview`.